File: trees/balancedBinaryTree.py
from trees.btf import *


class Solution:
    def isBalanced(self, root: TreeNode) -> bool:
        pass


if __name__ == '__main__':
    root = TreeNode(3)
    root.left = TreeNode(9)
    root.right = TreeNode(20)
    root.right.left = TreeNode(15)
    root.right.right = TreeNode(7)
    print("Input:")
    btf(root)
    print("Sulution:")
    print(Solution().isBalanced(root))

File: trees/cppRealizations/balancedBinaryTree.cpp
#include "btf.h"


class Solution {
public:
    bool isBalanced(TreeNode* root) {
        
    }
};


int main(){
    TreeNode* root = new TreeNode(1);
    root->left = new TreeNode(9);
    root->right = new TreeNode(20);
    root->right->left = new TreeNode(15);
    root->right->right = new TreeNode(7);
    cout << "Input:" << endl;
    btf(root);
    cout << "Solution:" << endl;
    cout << Solution().isBalanced(root) << endl;
    return 0;
}